src/CustomerView/DefaultCustomerView.php

```diff
@@ -110,7 +110,7 @@ public function getDetailviewData(CustomerInterface $customer)
         $vf = $this->viewFormatter;
 
         foreach ($definition->getFieldDefinitions() as $fd) {
-            if ($fd->getInvisible()) {
+            if ($fd->getInvisible() || $fd->getName() == 'password') {
```

Comment:
I would not exclude it only by name. I would exclude all PW type fields. 
We should never expose PW fields.

Reply:
```suggestion
            if ($fd->getInvisible() || $fd->getFieldtype() === 'password') {
```